canonicalwebteam/flask_base/app.py

```diff
@@ -28,6 +28,8 @@ def set_cache_control_headers(response):
         response.headers[
             "Cache-Control"
         ] = "public, max-age=300, stale-while-revalidate=360"
+    elif "v" in flask.request.args:
```

Comment:
This will cache anything with `v` on the query string for that amount of time, are we sure that doing this is safe?

Reply:
Yeah, this is why I asked in Mattermost. The last thing we want is one day to debug something like this. An extra condition to check if the request is for a static file would be neat but I don't know a good solution.

We could check if "." is present as static files use to have an extension, do you know another way?

Reply:
Wait, in all our flask apps the path for static files start with `/static/`, I will check that

Reply:
`flask.request.path.startswith("/static") and "v" in flask.request.args`?

Reply:
Yes, it's done like that now